src/common/require-module.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+// This module uses `eval("require")` to prevent tools such as Rollup
+// from hoisting the requires.
+
+// A babel plugin will look for `eval("require")()` and transform to
+// `require()` in the bundle, and rewrite the paths to
+// require from the top-level.
+
+// NOTE: separate const and function to do the following:
+// * execute the `eval` call just once
+// * increase the chance that an easy-to-understand function name
+//   shows up in case of a stack trace
+
+const requireModuleAtPath = eval("require");
+
+function requireModule(path) {
+  return requireModuleAtPath(path);
+}
```

Comment:
Or maybe try this

```js
function requireModule(path) {
  return eval("require")(path);
}
```

Reply:
For `require.resolve` we can do

```js
requireModule.resolve = eval("require").resolve 
```

Reply:
I tried it, did not seem to get `yarn prepare-release` to pass.

Reply:
It  shouldn't